superset/reports/notifications/email.py

```diff
@@ -94,6 +95,9 @@ def _get_content(self) -> EmailContent:
             html_table = ""
 
         call_to_action = __("Explore in Superset")
+        url = ""
+        if self._content.url is not None:
+            url = modify_url_query(self._content.url, standalone="0")
```

Comment:
Small nit, no need to fix because CI is slow.

```suggestion
        url = (
            modify_url_query(self._content.url, standalone="0")
            if self._content.url is not None
            else ''
        )
```